exercises/webapp/.storybook/preview.js

```diff
@@ -0,0 +1,4 @@
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
```

Comment:
Interesting solution! Why did you use Regex at this position?

Reply:
This is auto-generated code. It's responsible for the auto-magic event handlers like `onHandleWhatever` prefixed with `on`. I prefer to setup event handlers explicitly per storybook.